README.md

```diff
@@ -240,6 +240,24 @@ lgtm query \
 
 ---
 
+## Experimental commands
+
+### Get results from a query ID
+
+```bash
+lgtm x-list-query-results XXXXXXXXXXXXXXXXXXX
+```
+
+#### Examples
+
+##### Get projects name
+```bash
+lgtm x-list-query-results XXXXXXXXXXXXXXXXXXX | 
+jq '.[].Project.displayName' | tr -d '"'
```

Comment:
```suggestion
lgtm x-list-query-results XXXXXXXXXXXXXXXXXXX | jq -r '.[].Project.displayName'
```

Reply:
It would be better to provide examples that also use `--min-alerts` and `--min-results`, and retrieve the URL of the projects (usable for other stuff) instead of the name.

Reply:
Good idea, I'll make them later if you want ^^

Reply:
Something like:

Reply:
#### Examples

##### List project URLs of projects that have at least one result in the query run

```bash
lgtm x-list-query-results XXXXXXXXXXXXXXXXXXX --min-results=1 |  jq -r ".[].Project.externalURL.url"
```

##### List project URLs of projects that have at least one alert in the query run

```bash
lgtm x-list-query-results XXXXXXXXXXXXXXXXXXX --min-alerts=1 |  jq -r ".[].Project.externalURL.url"
```

Reply:
Added!